bot/seasons/evergreen/issues.py

```diff
@@ -3,55 +3,74 @@
 import discord
 from discord.ext import commands
 
-from bot.constants import Colours
+from bot.constants import Colours, Emojis
 from bot.decorators import override_in_channel
 
 log = logging.getLogger(__name__)
 
+ICONS = {"ISSUE": Emojis.issue,
+         "ISSUE_CLOSED": Emojis.issue_closed,
+         "PULL_REQUEST": Emojis.pull_request,
+         "PULL_REQUEST_CLOSED": Emojis.pull_request_closed,
+         "MERGE": Emojis.merge}
+
+RESP_VALUE = {404: "Issue/pull request not located! Please enter a valid number!",
+              403: "rate limit has been hit! Please try again later!"}
+
 
 class Issues(commands.Cog):
     """Cog that allows users to retrieve issues from GitHub."""
 
     def __init__(self, bot: commands.Bot):
         self.bot = bot
 
-    @commands.command(aliases=("issues",))
+    @commands.command(aliases=("pr",))
     @override_in_channel()
-    async def issue(
-        self, ctx: commands.Context, number: int, repository: str = "seasonalbot", user: str = "python-discord"
-    ) -> None:
-        """Command to retrieve issues from a GitHub repository."""
-        api_url = f"https://api.github.com/repos/{user}/{repository}/issues/{number}"
-        failed_status = {
-            404: f"Issue #{number} doesn't exist in the repository {user}/{repository}.",
-            403: f"Rate limit exceeded. Please wait a while before trying again!"
-        }
-
-        async with self.bot.http_session.get(api_url) as r:
-            json_data = await r.json()
-            response_code = r.status
+    async def issue(self, ctx: commands.Context, number: int, repository: str = "seasonalbot",
+                    user: str = "python-discord") -> None:
+        """Command to get issues/pull request from GitHub."""
+        url = f"https://api.github.com/repos/{user}/{repository}/issues/{str(number)}"
```

Comment:
```suggestion
        url = f"https://api.github.com/repos/{user}/{repository}/issues/{number}"
```
f-strings evaluate the integers to strings properly, thus there is no need to pass it to `str` first.